components/widgets/menu/menu.jsx

```diff
@@ -111,16 +111,18 @@ export default class Menu extends React.PureComponent {
         }
 
         return (
-            <ul
-                id={id}
-                className='a11y__popup Menu dropdown-menu'
-                ref={this.node}
-                style={styles}
-                role='menu'
-                aria-label={ariaLabel}
-            >
-                {children}
-            </ul>
+            <div className='a11y__popup Menu__container'>
```

Comment:
Not strictly against this, but normally we use the component uppercase name as class of the root node of the component, to allow other components affect it externally only knowing the component name. This approach is breaking that principle, so if somebody wants to affects the menu, probably is going to use `Menu` class name, which is going to affect a subtree of the component. My proposal would be use here `Menu` instead of `Menu_container` and in the `ul` use `Menu_content` instead of `Menu`

Reply:
Cool, can update that.